src/applescript/calendar.js

```diff
@@ -1,19 +1,20 @@
-const exec = require("./exec.js"),
-  { GET_ALL_EVENTS } = require("./scripts.js");
+const exec = require('./exec.js'),
```

Comment:
are you using prettier to format code? I prefer single quotes too but like spaces around braces. 

![CleanShot 2023-01-04 at 07 02 10](https://user-images.githubusercontent.com/1034890/210432514-7e301c42-b942-4559-bd1d-052ac33ecdd4.png)